Packages/com.unity.inputsystem/InputSystem/Actions/InputAction.cs

```diff
@@ -1027,6 +1063,38 @@ public unsafe object ReadValueAsObject()
             return null;
         }
 
+        /// <summary>
+        /// Read the current amount of actuation of the control that is driving this action.
+        /// </summary>
+        /// <returns>Returns the current level of control actuation (usually [0..1]) or -1 if
```

Comment:
The `magnitude` property can be -1 even though `haveMagnitude` is true. The `Actions_CanQueryMagnitudeFromAction_WithQuaternionControl_ReturnsInvalidMagnitude` test handles this scenario where the attitude Quaternion control is actuated but doesn't have a meaningful magnitude value, so it returns -1.

To me, it seemed like returning 0 in this method was preferable to returning -1 if no control is actuated, so that's why I used that value in this method for the default.

Reply:
ok